lib/matplotlib/testing/_markers.py

```diff
@@ -0,0 +1,48 @@
+"""
+pytest markers for the internal Matplotlib test suite.
+"""
+
+import logging
+import shutil
+
+import pytest
+
+import matplotlib.testing
+from matplotlib import _get_executable_info, ExecutableNotFoundError
+
+
+_log = logging.getLogger(__name__)
+
+
+def _checkdep_usetex():
+    if not shutil.which("tex"):
+        _log.warning("usetex mode requires TeX.")
+        return False
+    try:
+        _get_executable_info("dvipng")
+    except ExecutableNotFoundError:
+        _log.warning("usetex mode requires dvipng.")
+        return False
+    try:
+        _get_executable_info("gs")
+    except ExecutableNotFoundError:
+        _log.warning("usetex mode requires ghostscript.")
+        return False
+    return True
+
+
+needs_ghostscript = pytest.mark.skipif(
+    "eps" not in matplotlib.testing.compare.converter,
+    reason="This test needs a ghostscript installation")
+needs_lualatex = pytest.mark.skipif(
```

Comment:
@oscargus Sorry I missed this before merging, but these names seem not so optimal now?  needs_{lua,pdf,xe}latex now check both for the tex engine, *and* for the presence of pgf (this was less of an issue when the marks were local to test_backend_pgf.  Perhaps rename them to `needs_{lua,pdf,xe}tex_pgf`?  (Or just move them back to being local to the pgf backend test suite.)